webrender_traits/Cargo.toml

```diff
@@ -14,11 +14,11 @@ webgl = ["offscreen_gl_context"]
 app_units = "0.4"
 bincode = "1.0.0-alpha2"
 byteorder = "1.0"
-euclid = "0.13"
+euclid = "0.14.2"
```

Comment:
0.14.0  has the dreaded vector.to_point() bug (xyy). My understanding is that by specifying 0.14.2 I can get any version between 0.14.2 and 0.15.